examples/cifar10_to_monai.ipynb

```diff
@@ -0,0 +1,606 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 66,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class PILToNumpy(object):\n",
+    "    def __init__(self):\n",
+    "        pass\n",
+    "    \n",
+    "    def __call__(self, img):\n",
+    "        assert isinstance(img, Image.Image), \"input must be a PIL image.\"\n",
+    "\n",
+    "        return np.array(img).astype(np.float32)\n",
+    "\n",
+    "transform = transforms.Compose(\n",
+    "                [PILToNumpy(),\n",
+    "                 monai.transforms.AsChannelFirst(),\n",
+    "                 monai.transforms.RandFlip(prob=0.5, spatial_axis=[1, 0]),\n",
+    "                 monai.transforms.RandRotate(prob=0.5, degrees=90),\n",
+    "                 monai.transforms.IntensityNormalizer(np.array((0, 0, 0)), \n",
+    "                                                      np.array((255, 255, 255))),\n",
+    "                 monai.transforms.GaussianNoise(),\n",
+    "                 monai.transforms.ToTensor(),\n",
+    "                 transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's retrain the network and run it on test set."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Epoch 1/2, Iteration 2000/12500: Loss = 2.1631794204711916\n",
+      "Epoch 1/2, Iteration 4000/12500: Loss = 1.820360756099224\n",
+      "Epoch 1/2, Iteration 6000/12500: Loss = 1.6927146515846252\n",
+      "Epoch 1/2, Iteration 8000/12500: Loss = 1.5963873181343078\n",
+      "Epoch 1/2, Iteration 10000/12500: Loss = 1.510814044341445\n",
+      "Epoch 1/2, Iteration 12000/12500: Loss = 1.4680281994044782\n",
+      "Epoch 2/2, Iteration 2000/12500: Loss = 1.4026644861251116\n",
+      "Epoch 2/2, Iteration 4000/12500: Loss = 1.3909978227764368\n",
+      "Epoch 2/2, Iteration 6000/12500: Loss = 1.3315526320040225\n",
+      "Epoch 2/2, Iteration 8000/12500: Loss = 1.3111969945579767\n",
+      "Epoch 2/2, Iteration 10000/12500: Loss = 1.3066390281617641\n",
+      "Epoch 2/2, Iteration 12000/12500: Loss = 1.290063578262925\n",
+      "Finished Training in 116.47749829292297 seconds.\n"
+     ]
+    }
+   ],
+   "source": [
+    "train(no_epochs, trainloader, optimizer, net, criterion)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 68,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Accuracy of the network on the 10000 test images: 56 %\n"
```

Comment:
Seems the result is similar after you added several random transforms from MONAI, should we show these transforms obviously help the training?

Reply:
The only main addition is a random flip transform. CIFAR-10 is a very simple dataset. To show big accuracy increase, will need a more complicated dataset.